### PR TITLE
style error on checkbox, closes #135

### LINKED ISSUE #135: [UI] Align terms and conditions line with checkbox
When applying to a conference, the checkbox for the Terms and Conditions is not aligned with the text (see screenshot).

screen shot 2016-01-18 at 17 35 32

### PR DESCRIPTION
**Pull request by lislis**
There it is!

But please check if I merged the template correctly :P

### CODE REVIEW

**Comment by lisbethmarianne** [comment]
Somehow you added some extra form fields (the old ones), so now there is double name and email/email confirmation.

**Comment by lislis** [comment]
True! That was wrongly merged from my side, sorry. I fixed it and rebased.

File: app/views/applications/new.html.erb [hunk 1]
```diff
@@ -26,8 +26,13 @@
     <%= f.form_field :text_field, :email, 'Email' %>
     <%= f.form_field :text_field, :email_confirmation, 'Email Confirmation' %>
 
-    <% if @event.travel_funded? %>
-      <div class="form_field">
+    <div class="form_field">
```
Comment (lisbethmarianne) [inline app/views/applications/new.html.erb]: Hm, I think this also doesn't belong here anymore. We only have the visa checkbox now. 
```
<% if @event.travel_funded? %>
    <div class="form_field">
      <%= label_tag do %>
      <%= f.check_box :visa_needed %>
        I will need a visa to attend
      <% end %>
    </div>
  <% end %>
```
The problem with the checkbox is only on the terms and conditions, so I think you can leave it as it was before.

**Comment by lislis** [comment]
Ok, this should (finally) be fixed now

**Comment by lisbethmarianne** [comment]
Wohoooo! Looks perfect. Thanks so much!

**Pull request merged**